tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -823,6 +829,11 @@ def test_bounce(self, clean, connect_protocol, metadata_quorum, use_new_coordina
         metadata_quorum=[quorum.isolated_kraft],
         use_new_coordinator=[True, False]
     )
+    @matrix(
+        metadata_quorum=[quorum.isolated_kraft],
+        use_new_coordinator=[True],
+        group_protocol=consumer_group.all_group_protocols
+    )
```

Comment:
Either we need to add the `group_protocol` parameter, or else just remove this new `@matrix`.

```suggestion
```